src/index.html

```diff
@@ -1,18 +1,82 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__photo"
+        src="images/imac.jpeg"
```

Comment:
The `alt` attribute's value should be more descriptive of the image content, for example, 'Apple iMac 27" with Retina 5K display - front view'. This would improve accessibility and SEO.